src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="firstname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              max="100"
+              min="1"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Full date of birth:
+            <input type="date" name="dateOfBirth">
+          </label>
+        </div>
+        <div class="form__field last">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="acceptTheAgreement"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form__group">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="form__group">
```
add spaces between blocks of code for better readability

Reply:
thanks, i'll keep it in mind